packages/@azure/arm-applicationinsights-query/LICENSE.txt

```diff
@@ -1,21 +0,0 @@
-The MIT License (MIT)
```

Comment:
Remind me why we're deleting a package from the @azure folder?

Reply:
It's not a resource manager package so it's now "@azure/applicationinsights-query" package.